orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -712,11 +712,24 @@ class TaskController {
     return aStartTime <=> bStartTime ?: b.id <=> a.id
   }
 
+  static Boolean shouldReplace(Map.Entry entry, Map variables) {
```

Comment:
Why isn't this a primitive `boolean`?

Can we type the key & value of these maps & entries? At least to the point of a `String, Object`?

Reply:
Will fix both!